ocs_ci/ocs/ephernal_storage.py

```diff
@@ -0,0 +1,106 @@
+from logging import getLogger
+
+from ocs_ci.helpers.helpers import create_unique_resource_name
+from ocs_ci.ocs.constants import (
+    POD,
+    CEPHFS_INTERFACE,
+    RBD_INTERFACE,
+    CEPHBLOCKPOOL_SC,
+    ACCESS_MODE_RWO,
+    ACCESS_MODE_RWX,
+    CEPHFILESYSTEM_SC,
+)
+
+from ocs_ci.framework import config
+from ocs_ci.ocs.resources.pod import Pod
+
+log = getLogger(__name__)
+
+
+class EphernalPodFactory:
+
+    ephernal_pod = {
+        "kind": POD,
+        "apiVersion": "v1",
+        "metadata": {
+            "name": "pod_name1",
+            "namespace": config.ENV_DATA["cluster_namespace"],
+        },
+        "spec": {
+            "containers": [
+                {
+                    "name": "my-frontend",
+                    "image": "busybox:1.28",
+                    "volumeMounts": [
+                        {"mountPath": "/scratch", "name": "scratch-volume"}
+                    ],
+                    "command": ["sleep", "1000000"],
+                }
+            ],
+            "volumes": [],
+        },
+    }
```

Comment:
why are we writing pod template here it should be part of yaml inside ocs_ci/templates/app-pods